docs/readmes/nms/dev_testing.md

```diff
@@ -0,0 +1,29 @@
+---
+id: dev_testing
+title: Testing Tips
+hide_title: true
+---
+
+# Testing Tips
+
+This guide covers tips for quickly validating NMS changes.
+
+## Run Test on the host
+
+There are some simple tests that you should run after making changes.
+
+### Eslint
+Run `cd ${MAGMA_ROOT}/nms/app/ && yarn run eslint` to test that there are no linter errors.
+You can also run `cd ${MAGMA_ROOT}/nms/app/ && yarn run eslint --fix` to fix automatically
```

Comment:
Nit: don't break your lines. I.e. let each line just be as long as it needs to be